src/main/java/com/targomo/client/api/enums/MultiGraphAggregationType.java

```diff
@@ -25,10 +25,11 @@
     MATH                        (Constants.KEY_MULTIGRAPH_AGGREGATION_TYPE_MATH,                        false, false, false),
     GRAVITATION_HUFF            (Constants.KEY_MULTIGRAPH_AGGREGATION_TYPE_GRAVITATION_HUFF,            true,  true, false),
     LOGIT                       (Constants.KEY_MULTIGRAPH_AGGREGATION_TYPE_LOGIT,                       true,  true, false),
-    /**
-     * can only be used as main aggregation with no other aggregations in the pipeline - needs to be enabled for the endpoint
+     /**
+     * the following two aggregations can only be used as main aggregation with no other aggregations in the pipeline - needs to be enabled for the endpoint
      * **/
-    GRAVITATION_HUFF_OPTIMIZED  (Constants.KEY_MULTIGRAPH_AGGREGATION_TYPE_GRAVITATION_HUFF_OPTIMIZED,  false,  true, false);
+    GRAVITATION_HUFF_OPTIMIZED  (Constants.KEY_MULTIGRAPH_AGGREGATION_TYPE_GRAVITATION_HUFF_OPTIMIZED,  false,  true, false),
+    LOGIT_OPTIMIZED             (Constants.KEY_MULTIGRAPH_AGGREGATION_TYPE_LOGIT_OPTIMIZED,             true,   true, false);
```

Comment:
can you please set the value of "mayIgnoreOutliers" to false

Reply:
Comment addressed.